protocol/protocol.go

```diff
@@ -40,20 +40,26 @@ func (p *Manager) Stop() error {
 func newRouterInitializer(netAdapter *netadapter.NetAdapter, dag *blockdag.BlockDAG) netadapter.RouterInitializer {
 	return func() (*netadapter.Router, error) {
 		router := netadapter.NewRouter()
-		err := router.AddRoute([]string{wire.CmdTx}, startDummy(netAdapter, router, dag))
+		err := router.AddRoute([]string{wire.CmdPing, wire.CmdPong}, startPing(netAdapter, router, dag))
 		if err != nil {
 			return nil, err
 		}
 		return router, nil
 	}
 }
 
-func startDummy(netAdapter *netadapter.NetAdapter, router *netadapter.Router,
+// TODO(libp2p): Remove this and change it with a real Ping-Pong flow.
+func startPing(netAdapter *netadapter.NetAdapter, router *netadapter.Router,
```

Comment:
Can you undo changes to avoid redundant conflicts?

Reply:
I kinda want the PingPong flow to be there so that we can test connections until the real ping-pong flow is implemented